test-it/karma/javascriptinvoke/Reinitialize.Test.js

```diff
@@ -0,0 +1,43 @@
+describe('A JavaScript function invoke', function () {
+    'use strict';
+    // Reference aliases
+    var vireoHelpers = window.vireoHelpers;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo;
+
+    var jsComplexFunctionsViaUrl = fixtures.convertToAbsoluteFromFixturesDir('javascriptinvoke/ReinitializeToDefault.via');
+
+    beforeAll(function (done) {
+        fixtures.preloadAbsoluteUrls([
+            jsComplexFunctionsViaUrl
+        ], done);
+    });
+
+    beforeEach(async function () {
+        vireo = await vireoHelpers.createInstance();
+
+        window.NI_InternalFunctionReinitialize = function (viName, path) {
+            var valueRef = vireo.eggShell.findValueRef(viName, path);
+            vireo.eggShell.reinitializeToDefaultData(valueRef);
+        };
+    });
+
+    afterEach(function () {
+        // Cleanup functions
+        window.NI_ConcatenateValue = undefined;
+    });
+
+    it('with no parameters succesfully works', function (done) {
+        var runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, jsComplexFunctionsViaUrl);
```

Comment:
I think using the JavaScriptInvoke is adding unnecessary complexity to the test.

It should be structured like:
```
it {
reebootandloadvia
run the via to completion
expect preconditions before reinit is called
execute the reinitialize to default api function
expect postconditions after reinit is called
}
```

For new tests we should also be using the [async based runner](https://github.com/ni/VireoSDK/blob/master/test-it/karma/systemlogging/SystemLogging.Test.js#L33). Note that the `it` statement is an `async function`, that we call `await runSlicesAsync()`, and that we use let / const instead of var. (Vireo has a couple steps left in its es6 transition :P)